deploy/deployFactoryAccount.ts

```diff
@@ -0,0 +1,25 @@
+import { utils, Wallet } from "zksync-ethers";
+import * as ethers from "ethers";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { Deployer } from "@matterlabs/hardhat-zksync-deploy";
+
+export default async function (hre: HardhatRuntimeEnvironment) {
+  // Private key of the account used to deploy
+  const wallet = new Wallet("3344e420d69a700a94a808e7c429b4a8e85dc46c16e4a0ef84f35e9422f88771");
```

Comment:
Should read private key from the env. Also please make sure you don't use it anywhere in mainnet!!